### PR TITLE
prefix macro VERSION

### PR DESCRIPTION
**Pull request by cspiel1**
Avoids name conflict in zephyr

### CODE REVIEW

**Comment by cspiel1** [comment]
On Windows the installation of OpenSSL fails. This has nothing to do with this PR.

**Comment by sreimers** [comment]
Is fixed in main

File: src/http/request.c [hunk 1]
```diff
@@ -31,8 +31,8 @@
 #define DEBUG_LEVEL 5
 #include <re_dbg.h>
 
-#ifndef VERSION
-#define VERSION "???"
+#ifndef RE_VERSION
+#define RE_VERSION "???"
```
Comment (alfredh) [inline src/http/request.c:35]: The version string is re-defined here ?

Comment (sreimers) [reply src/http/request.c:35]: Yes this looks strange, better to include `re_sys.h`

Comment (cspiel1) [reply src/http/request.c:35]: Yes, I add a commit.

**Pull request merged**